src/main/java/org/diatliuk/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -1,28 +1,42 @@
 package org.diatliuk.bookstore.service.impl;
 
 import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.diatliuk.bookstore.dto.BookDto;
+import org.diatliuk.bookstore.dto.CreateBookRequestDto;
+import org.diatliuk.bookstore.exception.EntityNotFoundException;
+import org.diatliuk.bookstore.mapper.BookMapper;
 import org.diatliuk.bookstore.model.Book;
 import org.diatliuk.bookstore.repository.BookRepository;
 import org.diatliuk.bookstore.service.BookService;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
+@RequiredArgsConstructor
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
 
-    @Autowired
-    public BookServiceImpl(BookRepository bookRepository) {
-        this.bookRepository = bookRepository;
+    @Override
+    public BookDto save(CreateBookRequestDto requestDto) {
+        Book book = bookMapper.toModel(requestDto);
+        return bookMapper.toDto(bookRepository.save(book));
     }
 
     @Override
-    public Book save(Book book) {
-        return bookRepository.save(book);
+    public List<BookDto> findAll() {
```

Comment:
Be consistent in naming, if you called one service method with `get` pattern, try to use it in other methods too.